projects/storefrontapp-e2e-cypress/cypress/helpers/user.ts

```diff
@@ -0,0 +1,12 @@
+export function generateMail(alias: string, newTimestamp: boolean) {
```

Comment:
I think that not only `requireLoggedIn` function deserves the documentation, but also this function, as it will be used to create shared new users by developers (and understand how already shared users are created). I think it's not obvious when and how the `newTimestamp` option should be used.

Reply:
Wrote a paragraph about that in docs file